recommenders/models/rbm/rbm.py

```diff
@@ -14,6 +16,9 @@ class RBM:
 
     def __init__(
         self,
+        n_users,
```

Comment:
Maybe you could avoid `n_users` in the constructor of the model. As far as I can see it is used only inside `fit()` for the minibatch, isn't it? Is there a scenario in which `n_users` should be other than the number of users in the training data? If not, then you could remove `n_users` from the constructor and compute the minibatch inside `fit()`. 
This would also make the model simpler to handle for the end user, since now if you want to retrain with more data you need to do something like
```
model = RBM(n_users=10000)
model.fit(Xtr)
Xtr2 = pd.concat([Xtr, Xtr_new]) 
model = RBM(n_users=15000)
model.fit(Xtr2)
```
Without `n_users` in `__init()__` you would avoid the second call to `__init()__`. It may also cause some problems with save and load.

Reply:
@pradnyeshjoshi FYI in case you didn't see this

Reply:
@miguelgfierro I have created a [PR](https://github.com/microsoft/recommenders/pull/1622) to remove `n_users` from the constructor. Looks like Andreas has approved it, you may merge it into staging if all looks good.